examples/module/client.py

```diff
@@ -9,9 +9,9 @@
 
 
 async def connect():
-    creds = Credentials(type="<your authentication type here>", payload="<your authentication payload here>")
+    creds = Credentials(type="robot-location-secret", payload="znm8jwc5ossauwmy24x7b9ts7ptb750ajg8bzmxv9utkoic2")
```

Comment:
We should not commit an actual robot uri or payload.

Reply:
Ah thank you for catching that!